io.github.ergolyam.Drosophila.yml

```diff
@@ -0,0 +1,390 @@
+app-id: io.github.ergolyam.Drosophila
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.freedesktop.Sdk//24.08
```

Comment:
not sure what do you are reading but this is wrong and I doubt it will work. and if it doesn't then you don't need the gnome runtime.